src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,107 @@
+package duke.parser;
+
+import duke.DukeException;
+import duke.tasks.Task;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Parser is the class that parses raw user commands and executes the intended effect
```

Comment:
Maybe end with a full stop for consistency for the description of classes and methods?